pithos/plugins/notification_icon.py

```diff
@@ -112,6 +79,18 @@ def check_for_plasma():
                 None
             )
 
+        def on_StatusNotifierWatcher_finish(source, result, data):
+            try:
+                StatusNotifierWatcher = Gio.DBusProxy.new_finish(result)
+            except GLib.Error as e:
+                logging.debug(e)
+                check_for_x()
+            else:
+                if StatusNotifierWatcher.get_name_owner():
+                    init_appindicator()
+                else:
+                    check_for_plasma()
```

Comment:
I still may give it a shot for the challenge and novelty of it but I really don't see the actually usefulness of it. I'm fine not supporting tray icons in KDE Plasma outside of the Buntu's for now.

Your thoughts @TingPing?